specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json

```diff
@@ -6915,6 +6915,18 @@
             "name": "VirtualMachinePriorityTypes",
             "modelAsString": true
           }
+        },
+        "evictionPolicy": {
+          "type": "string",
+          "description": "Specifies the eviction policy for virtual machines in the low priority scale set. <br><br>Minimum api-version: 2017-10-30-preview",
+          "enum": [
+            "Deallocate",
+            "Delete"
+          ],
+          "x-ms-enum": {
+            "name": "VirtualMachineEvictionPolicyTypes",
+            "modelAsString": false
```

Comment:
Marking `modelAsString` as false can cause breaking changes if new enum types are added. Please reconsider this.

Reply:
Ok, I will change it to true then

Reply:
Done